src/containers/SignInSide/index.jsx

```diff
@@ -66,14 +61,14 @@ function SignInSide() {
               </Grid>
               <Grid item>
                 <Link href="#" variant="body2">
-                  Don't have an account? Sign Up
+                  Don`t have an account? Sign Up
```

Comment:
I think the right way to write this sentence is `Don't have an account? Sign Up`

Reply:
I've changed this because ESLint was accusing an error, but I'll try another solution.